firestore/query.go

```diff
@@ -141,15 +171,15 @@ func (q Query) OrderBy(path string, dir Direction) Query {
 		q.err = err
 		return q
 	}
-	q.orders = append(q.copyOrders(), order{fp, dir})
+	q.orders = append(q.copyOrders(), order{fp, nil, dir})
```

Comment:
nit: If this new field was added to the end of the struct no code change would be needed here. In general I would avoid using unkeyed struct literals where possible as they can lead to more trouble than they are worth, imo.

Reply:
I agree with this, though adding a fieldRef to the end of a set of things (fieldPath, direction, fieldRef) felt a bit off and since it is internal I could safely refactor. I could alter these to all state the field name though if that is preferred. It would like make it more readable anyway.